src/containers/tud/constants/JsonKeyAnswerIdMapping.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+import { PROPERTY_CONSTS } from './SchemaConstants';
+
+export default {
+  adult_media: PROPERTY_CONSTS.ADULT_MEDIA,
```

Comment:
Why don't you use the translation keys here? Seems like a bug waiting to happen.

```
import TranslationKeys from './TranslationKeys';

export default {
  [TranslationKeys.ADULT_MEDIA]: PROPERTY_CONSTS.ADULT_MEDIA,
  ...
};
```